es6/README.md

```diff
@@ -48,9 +80,200 @@ const MaxAllowed = 7;
 const maxAllowed = 7;
 ```
 
+Factor out constants into a constants file if a given [module](#modules) has more than 3 constants:
+
+```js
+// good (imports constants from constants file)
+import * as constants from './constants';
```

Comment:
Maybe add some direction in regards to importing. Such as when there are so many individual imports from one file that `* as constants` should be used rather than importing the individual things?